src/osd/scrubber/scrub_backend.cc

```diff
@@ -59,7 +59,7 @@ ScrubBackend::ScrubBackend(ScrubBeListener& scrubber,
     , m_conf{m_scrubber.get_pg_cct()->_conf}
     , clog{m_scrubber.get_logger()}
 {
-  m_formatted_id = m_pg_id.calc_name_sring();
+  m_formatted_id = m_pg_id.calc_name_string();
```

Comment:
I'm reluctantly inclined to accept since the invocations are also changed.  @epuertat  ?

Reply:
👍🏼